### ISSUE TITLE
Datepicker for article dates

### ISSUE THREAD

**Issue by JGilbert-eLife** [substance/texture#980]
We probably need a datepicker option on the date fields in article information. At the moment, it is unclear what format is required and whatever is entered is being dropped into the XML. In this example, none of what I entered could be classed as an iso-8601-date value! We are also missing any content for these dates; there are just elements with attributes.

![screen shot 2018-10-19 at 15 57 45](https://user-images.githubusercontent.com/1785305/47226160-bc835680-d3b7-11e8-8160-d7c5bdaff0b2.png)

**Comment by michael** [comment]
Yeah we should talk about this and normalise across the user interface. What may be tricky is allowing different formats. Like year-only, year + season, year + month, year+month+day. It's not only a UI issue but an effort to normalise date usage in XML. Currently there's a lot mixed up. And sometimes we have human readable dates in addition to iso-8601 values.

Think we should dedicate a few days of specification and development to this topic.
